### PR TITLE
add GMK Vaporwave

### PR DESCRIPTION
**Pull request by akatopo**
Adds vaporwave colorway. Uses colors from this [GH thread](https://geekhack.org/index.php?topic=99875.0).
Had to lighten the original pink color and change the font weight to bold as the contrast was very bad (see screenshots below). Coincidentally it looks more like the real thing 😄 

orig-pink
lighter-pink

### CODE REVIEW

**Comment by yanfali** [comment]
Needs permission from the original designer. I'll leave it to @noroadsleft to work with you about the colors. Honestly, it doesn't look very good on screen tbh. It works much better as a physical object. legibility is more important than color accuracy for a UI.

**Comment by akatopo** [comment]
Of course the PR(#843) that I tried to emulate was from a designer 😅. Asked NP for permission on GH:

permission proof

Tried tweaking the pink a bit more, leaving a screenshot

Screen Shot 2020-12-02 at 1 27 58 PM

**Comment by noroadsleft** [comment]
> Screen Shot 2020-12-02 at 1 27 58 PM

This looks pretty good, but doesn't seem to match the current state of this PR. Did you forget to `git push`?

**Comment by akatopo** [comment]
> > Screen Shot 2020-12-02 at 1 27 58 PM
> 
> This looks pretty good, but doesn't seem to match the current state of this PR. Did you forget to `git push`?

just waiting on feedback, pushed :-)

File: src/scss/colorways.scss
Comment (noroadsleft) [inline src/scss/colorways.scss:1174]: Color shift this the other way to increase the contrast.

```suggestion
    background: darken(#6244bb, 10%);
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
I'll probably want to adjust the colors further later on to improve the contrast, but this will do for now.

Thanks!

**Pull request merged**
